TestSuite/integration.py

```diff
@@ -59,6 +62,8 @@ def build(
         description: Optional[str] = None,
         changelog: Optional[str] = None,
         image: Optional[bytes] = None,
+        commands_txt: Optional[str] = None,
+        test: Optional[str] = None,
     ):
```

Comment:
why in this pr?

Reply:
because i needed it to create some examples of content items that i wanted to load and check, do you think it should be added in a different PR? its was mainly added for UTs.